File: app/src/main/java/dev/ragnarok/fenrir/api/model/VKApiConversationMembers.java
package dev.ragnarok.fenrir.api.model;

import com.google.gson.annotations.SerializedName;

public class VKApiConversationMembers {
    @SerializedName("member_id")
    public int member_id;

    @SerializedName("invited_by")
    public int invited_by;

    @SerializedName("join_date")
    public long join_date;

    @SerializedName("is_admin")
    public boolean is_admin;

    @SerializedName("is_owner")
    public boolean is_owner;

    @SerializedName("can_kick")
    public boolean can_kick;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/VKApiConversationMembers.kt
package dev.ragnarok.fenrir.api.model

import com.google.gson.annotations.SerializedName

class VKApiConversationMembers {
    @SerializedName("member_id")
    var member_id = 0

    @SerializedName("invited_by")
    var invited_by = 0

    @SerializedName("join_date")
    var join_date: Long = 0

    @SerializedName("is_admin")
    var is_admin = false

    @SerializedName("is_owner")
    var is_owner = false

    @SerializedName("can_kick")
    var can_kick = false
}